components/Text.js

```diff
@@ -39,6 +40,8 @@ Span.defaultProps = {
   lineHeight: 'inherit',
 };
 
+export const Div = P.withComponent('div');
```

Comment:
Not a fan of this, `div` doesn't sounds like a text component (aka something that has to do with typography).

If you need to write a div with text styles, you can also write:

```jsx
<P as="div" fontSize="14px">Text</P>
```